outbound-dialer/src/main/java/org/motechproject/nms/outbounddialer/domain/AuditRecord.java

```diff
@@ -8,35 +8,62 @@
 
 @Entity(tableName = "nms_obd_audit_records")
 public class AuditRecord {
+    /**
+     * The identifier field is used in each targetFile csv row combined with the subscription id to form a uniquely
+     * identifying id for a specific OBD record
+     */
     @Field
     @UIDisplayable(position = 0)
-    private FileType type;
+    private String identifier;
```

Comment:
Not sure if I understand the comments above. The identifier here is then combined with the subscription id to make the unique id? is this the batch id or request id that IMI wants?